### DISCUSSION TITLE
💋 【共学营打卡】【第5次 Milestone】给 scaffold-move 提交任意有意义的 PR

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#194] [Projects-Others]
scaffole-move:

> https://github.com/NonceGeek/scaffold-move

**Comment by MagicGordon** [comment]
https://github.com/NonceGeek/scaffold-move/pull/4

**Comment by v1xingyue** [comment]
PR 列表如下：
https://github.com/NonceGeek/scaffold-move/pulls?q=is%3Apr+author%3Av1xingyue

**Comment by livelybug** [comment]
https://github.com/NonceGeek/scaffold-move/pulls?q=is%3Apr+author%3Alivelybug
应该算数吧。。。

**Comment by xiegudong45** [comment]
https://github.com/NonceGeek/scaffold-move/pull/9 增加了一个删除功能

**Comment by baidang201** [comment]
https://github.com/NonceGeek/scaffold-move/pull/19  eslint
